setup.cfg

```diff
@@ -1,4 +1,4 @@
-[pytest]
+[tool:pytest]
```

Comment:
I'm not entirely sure when the `tool:pytest` syntax was supported by pytest, but it breaks for me when using the old syntax with pytest 4. If there's no way to maintain compatibility between 3/4 here, might as well go back to a pytest.ini

Reply:
been supported for a long time iirc so this should be totally fine

Reply:
JFI: pytest 4.4 will not error out anymore with `[tool:pytest]` and `[pytest]` - https://github.com/pytest-dev/pytest/pull/5008.